pages/Addons.md

```diff
@@ -48,6 +48,7 @@ Tinkers (WIP) | Skizzles | <details><summary>expand</summary>An addon for Slimef
 MoreTools (WIP) | LinoxGH | <details><summary>expand</summary>This addon adds more tools to Slimefun. These tools are meant to improve your gameplay experience, like Crescent Hammer(Wrench) and Cargo Copier.</details> | N/A | [GitHub](https://github.com/LinoxGH/MoreTools) | [Builds](https://thebusybiscuit.github.io/builds/LinoxGH/MoreTools/build/)
 FluffyMachines | NCBPFluffyBear | <details><summary>expand</summary>This addon adds early and late game machines, generators, and items to make automation more fun. Among these are the Auto Crafting Table, Auto Ancient Altar, and Ender Chest Cargo Nodes.</details> | [Readme](https://github.com/NCBPFluffyBear/FluffyMachines/blob/master/README.md) | [Github](https://github.com/NCBPFluffyBear/FluffyMachines) | [Builds](https://thebusybiscuit.github.io/builds/NCBPFluffyBear/FluffyMachines/master/)
 InfinityExpansion | Mooy1 | <details><summary>expand</summary>This addon aims to add a larger endgame to slimefun by adding 'Infinity ingots' which take an extreme amount of resources to create, and are used to craft much stronger machines. It is somewhat based of of the mod 'avaritia'.</details> | [Readme](https://github.com/Mooy1/InfinityExpansion/blob/Dev/README.md) | [Github](https://github.com/Mooy1/InfinityExpansion) | [Builds](https://thebusybiscuit.github.io/builds/Mooy1/InfinityExpansion/master/)
+FoxyMachines | GallowsDove | <details><summary>expand</summary>This addon adds whatever I feel like adding, including weapons, tools and machines. It currently serves as a coding practice. </details> | [Readme](https://github.com/GallowsDove/FoxyMachines/blob/master/README.md) | [Github](https://github.com/GallowsDove/FoxyMachines) | [Releases](https://github.com/GallowsDove/FoxyMachines/releases)
```

Comment:
You should add a proper description to the addon, like examples for what it actually adds.